src/coprocessor/endpoint.rs

```diff
@@ -792,6 +816,42 @@ mod tests {
         assert!(!resp.get_other_error().is_empty());
         assert_eq!(resp.get_other_error(), super::OUTDATED_ERROR_MSG);
     }
+
+    #[test]
+    fn test_exec_details_with_long_query() {
```

Comment:
Is it necessary since https://github.com/pingcap/tikv/pull/2638/files#diff-c1fcd538536072b17fa388c7722dbc17R2145 cover it already.